src/test/java/Homework16.java

```diff
@@ -0,0 +1,29 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class Homework16 extends BaseTest {
+    @Test
+    public static void RegistrationNavigation() {
+
+        WebDriver driver = new ChromeDriver();
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+
+        String url = "https://bbb.testpro.io/";
+        driver.get(url);
+
+        WebElement registrationLink = driver.findElement(By.cssSelector("[id='hel']"));
+        registrationLink.click();
+
+        String registrationUrl = "https://bbb.testpro.io/registration.php";
+        Assert.assertEquals(driver.getCurrentUrl(), registrationUrl);
```

Comment:
Please, add Homework16 in TestNG.xml  run tests

For example

![image](https://user-images.githubusercontent.com/104761482/221413414-f7cf3e5e-b2a0-4547-ad0e-fb50d93eb237.png)

Reply:
You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/221780957-6cec2578-6380-4b3b-8576-286bd0f7d3ef.png)